assets/app/scripts/services/logger.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+angular.module('openshiftConsole')
+.provider('Logger', function() {
+  this.$get = function() {
+    var OSLogger = Logger.get("OpenShift");
```

Comment:
comment that this is using the global `Logger` object... it's a little confusing... I thought it was using the object defined by this service